### PR TITLE
[Focus Mode] New DevTools Memory Problems Docs

### PR DESCRIPTION
**Pull request by JLSeto**
Updates to screenshots

### CODE REVIEW

**Review by captainbrosset — CHANGES_REQUESTED** [review]
Leaving a bunch of comments about images.
Also one general comment: there are multiple places in these articles where we say "Memory panel" instead of "Memory tool". We might want to take this opportunity to fix this too.

Just reiterating that I'm not necessarily expecting you to address these comments. I can do this once we get near to merging the Focus Mode PRs.
Thanks for the work on this.

File: microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots-images/memory-problems-gh-nodejs-benchmarks-run-memory-heap-snapshots-constructor-highlight.png
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots-images/memory-problems-gh-nodejs-benchmarks-run-memory-heap-snapshots-constructor-highlight.png:1]: rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots?branch=pr-en-us-2709#constructor-group-entries-in-the-heap-profiler

It's the image in that section.

File: microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler-images/memory-problems-memory-allocation-timeline-snapshot-highlighted.png
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler-images/memory-problems-memory-allocation-timeline-snapshot-highlighted.png:1]: Rendered article section that contains the png:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler?branch=pr-en-us-2709#how-allocation-instrumentation-on-timeline-works

Remove Note formatting below the png; it reduces communication of the simple text that it contains.  per:
https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?toc=%2Fhelp%2Fplatform%2Ftoc.json&bc=%2Fhelp%2Fplatform%2Fbreadcrumb%2Ftoc.json&branch=main#alerts-note-tip-important-caution-warning

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Completed a Writer/Editor pass.  Briefly checked existing review comments.

File: microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler-images/memory-problems-memory-allocation-timeline-snapshot-highlighted.png
[the remark this answers is not in the document]
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/memory-problems/allocation-profiler-images/memory-problems-memory-allocation-timeline-snapshot-highlighted.png:1]: Keeping the image for now, and adding some blue bars. I'm also removing the Note formatting.
But we should also add some description for the image as you suggested too.

**Pull request merged**